tests/setup.coffee

```diff
@@ -1,19 +1,33 @@
+_ = require('lodash')
 Promise = require('bluebird')
-global.Promise = Promise
-require('isomorphic-fetch')
 
-_ = require('lodash')
+IS_BROWSER = window?
+
+if IS_BROWSER
+	# The browser mock assumes global fetch prototypes exist
+	# Can improve after https://github.com/wheresrhys/fetch-mock/issues/158
+	realFetchModule = require('fetch-ponyfill')({ Promise })
+	global.Headers = realFetchModule.Headers
```

Comment:
NIT: Can also be:
`_.assign(global, _.pick(realFetchModule, 'Headers', 'Request', 'Response'))`